src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1156,7 +1156,8 @@ def _configure_security_policy_storage_params(arg_ctx):
 
         c.argument('tier',
                    arg_type=tier_param_type,
-                   help='The edition component of the sku. Allowed values: GeneralPurpose, BusinessCritical.')
+                   help='The edition component of the sku. Allowed values include:'
```

Comment:
please use get_enum_types() for allowed values.

Reply:
Done. I've done the same thing for family too